src/main/drivers/serial.h

```diff
@@ -68,6 +70,23 @@ typedef struct serialPort_s {
     serialReceiveCallbackPtr rxCallback;
 } serialPort_t;
 
+#if defined(USE_SOFTSERIAL1) || defined(USE_SOFTSERIAL2)
```

Comment:
SERIAL_PORT_MAX_INDEX -> SERIAL_PORT_MAX_COUNT

Can't touch the RESOURCE_SOFT_OFFSET now, may be in 3.2.